### ISSUE TITLE
Store route creating new graph delay

### ISSUE THREAD

**Issue by vojtechmusilek** [oxigraph/oxigraph#237]
Hi I'm saving .ttl files with C# using this url `http://localhost:7878/store?graph=test`

When the graph 'test' is not created in Oxigraph it takes almost exactly 1 minute to get response, I think it has nothing to do with the C# HttpClient im using because that has timeout of 1min40s. But I may be wrong. Result code is 201-Created.

After this the graph is created in Oxigraph and other requests to store ttls are fast with result code 204-NoContent as expected. 

Any idea why is the first request taking so long?

**Comment by Tpt** [comment]
Hi! Thank you for trying Oxigraph.

A 1mn response time seems very slow indeed. How many triples are you inserting when creating the graph? Are you using a reasonably fast file system (i.e. not something like a bad NFS)? Which OS are you using? I am mostly testing on Linux and never had such bad response times on this operation with only a few inserted triples.

**Comment by vojtechmusilek** [comment]
Inserting 1 triple. Tried it on fast remote Linux and local Windows 11, both running Oxigraph in docker (using this image: ghcr.io/oxigraph/oxigraph:v0.3.5). 

I tried the same request in Postman and I don't understand the behaviour. It almost instantly returns 201-Created but then the time continues to count, like the connection is still open or something. Consistently right after 1minute it finishes and returns this body:
```
HTTP/1.1 500 Internal Server Error
content-type: text/plain; charset=utf-8
server: Oxigraph/0.3.5
content-length: 46

Resource temporarily unavailable (os error 11)
```

**Comment by Tpt** [comment]
Thank you for your detailed reply! I have reproduced the bug. It's a bug in the [http library used by Oxigraph](https://github.com/oxigraph/oxhttp).

When using curl in verbose mode, I get the warning `no chunk, no close, no size. Assume close to signal end`.

**Comment by Tpt** [comment]
I have just released v0.3.6 that should fix this bug. The artifacts (docker images...) are currently building and should be available in a few minutes.

**Comment by vojtechmusilek** [comment]
Thanks for the quick fix, I just tested it and it works.
